basics/github.md

```diff
@@ -11,13 +11,16 @@ In the admin settings for repos on GitHub, the recommended settings are:
     - require status checks to pass before merging
     - require that branches are up to date
 * Allow merge commits (`Options > Merge button`)
+* **Optional** Disable squash commits (`Options > Merge button`)
```

Comment:
I'd suggest we make it either `Recommended` or a new `Requirement` to disable squash commits. EngX never uses this feature and had it disabled explicitly in the repos we use most often and have remembered to change this.